File: back_end_fg/WebApplication1/WebApplication1/Models/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WimsApiMKI.Models
{
    public class Game
    {
        private int _id;
        private string _gameName;
        private string _gameImage;
        private bool _active;

        #region GetSet
        public int Id
        {
            get { return _id; }
            set { _id = value; }
        }

        public string GameName
        {
            get { return _gameName; }
            set { _gameName = value; }
        }

        public string GameImage
        {
            get { return _gameImage; }
            set { _gameImage = value; }
        }

        public bool Active
        {
            get { return _active; }
            set { _active = value; }
        }
        #endregion
    }
}

File: WIMS-front/src/app/Observables/Game.ts
export class Game {
    constructor(
      private id: number,

    ) {}
    private _GameName: string;
    private _GameImage: string;
    private _Active: boolean;

    get Id(): number {
        return this.id;
    }

    set Id(value: number) {
        this.id = value;
    }

    get GameName(): string {
        return this._GameName;
    }

    set GameName(value: string) {
        this._GameName = value;
    }

    get GameImage(): string {
        return this._GameImage;
    }

    set GameImage(value: string) {
        this._GameImage = value;
    }

    get Active(): boolean {
        return this._Active;
    }

    set Active(value: boolean) {
        this._Active = value;
    }
}
